src/t8_schemes/t8_standalone/t8_standalone_implementation.hxx

```diff
@@ -745,7 +796,28 @@ struct t8_standalone_scheme
   element_get_first_descendant_face (const t8_element_t *elem, const int face, t8_element_t *first_desc,
                                      const t8_element_level level) noexcept
   {
-    SC_ABORT ("This function is not implemented in this scheme yet.\n");
+    T8_ASSERT (element_is_valid (elem));
+    const t8_standalone_element<TEclass> *el = (const t8_standalone_element<TEclass> *) elem;
+    t8_standalone_element<TEclass> *first_descendant = (t8_standalone_element<TEclass> *) first_desc;
+
+    first_descendant->level = level;
+    if constexpr (T8_ELEMENT_NUM_EQUATIONS[TEclass]) {
+      SC_ABORT ("Only implemented for hypercubes.\n");
+    }
+    for (int idim = 0; idim < T8_ELEMENT_DIM[TEclass]; idim++) {
+      first_descendant->coords[idim] = el->coords[idim];
+    }
```

Comment:
first_descendant->coords is a std::array. You could also use a stl algorithm to copy the coords.